src/kubectl.ts

```diff
@@ -237,7 +237,7 @@ async function fromLines(context: Context, command: string): Promise<Errorable<{
     if (shellResult.code === 0) {
         let lines = shellResult.stdout.split('\n');
         lines = lines.filter((l) => l.length > 0);
-        const parsedOutput = parseLineOutput(lines);
+        const parsedOutput = parseLineOutput(lines, POD_OUTPUT_PARSING_REGEX);
```

Comment:
`fromLines` shouldn't be pod-specific.  Would it be correct to say `KUBECTL_OUTPUT_PARSING_REGEX`? (or `KUBECTL_OUTPUT_COLUMN_SEPARATOR`)?